.gitignore

```diff
@@ -5,3 +5,4 @@ darwin-amd64
 *.tar.gz
 linux-amd64
 *.tgz
+testdep/oauth2clientweb/vendor/**
```

Comment:
Would you mind removing this line from the ignore file, as we do want to include the vendor directory for better reliability & reproducibility of builds 😉